packages/wrangler/src/cli.ts

```diff
@@ -11,6 +13,8 @@ import type { UnstableDevWorker, UnstableDevOptions } from "./api";
  */
 if (typeof jest === "undefined" && require.main === module) {
 	main(hideBin(process.argv)).catch((e) => {
+		// Dump debug logs on crash
+		writeFileSync("wrangler-error.log", logger.debugLogs.join("\n"));
```

Comment:
I think we should write a message to stderr as well here saying that we have dumped this file and where to find it - the user might not be aware of the current working directory if Wrangler is triggered by another process.

Reply:
Now it writes `Writing debug logs to /Users/glen/src/work/d1-worker/test/manual/wrangler-error.log` (full path in case the user isn't running it directly)